flake/dev/generate-all-maintainers/extract-maintainers-meta.nix

```diff
@@ -0,0 +1,67 @@
+# Extract maintainers from nixvim configuration using meta.maintainers
+# This script evaluates an empty nixvimConfiguration and extracts the merged maintainer information
+let
+  nixvim = import ../../..;
+  lib = nixvim.inputs.nixpkgs.lib.extend nixvim.lib.overlay;
+
+  emptyConfig = lib.nixvim.evalNixvim {
+    modules = [ ];
+    extraSpecialArgs = { };
+    system = "x86_64-linux";
+  };
+
+  inherit (emptyConfig.config.meta) maintainers;
+
+  extractMaintainerObjects =
+    maintainerData:
+    lib.pipe maintainerData [
+      lib.attrValues
+      lib.concatLists
+      lib.unique
+    ];
+
+  allMaintainerObjects = extractMaintainerObjects maintainers;
+
+  allMaintainerNames = lib.filter (name: name != null) (
+    map (maintainer: maintainer.github) allMaintainerObjects
+  );
+
+  nixvimMaintainers = import ../../../lib/maintainers.nix;
+  nixvimMaintainerNames = lib.attrNames nixvimMaintainers;
+  partitionedMaintainers = lib.partition (nameValue: lib.elem nameValue.name nixvimMaintainerNames) (
+    lib.attrsToList maintainerDetails
+  );
+
+  maintainerDetails = lib.pipe allMaintainerObjects [
+    (map (obj: {
+      name = obj.github;
+      value = obj // {
+        source = if lib.elem obj.github nixvimMaintainerNames then "nixvim" else "nixpkgs";
+      };
+    }))
+    lib.listToAttrs
+  ];
+
+  categorizedMaintainers = {
+    nixvim = lib.listToAttrs partitionedMaintainers.right;
+    nixpkgs = lib.listToAttrs partitionedMaintainers.wrong;
+  };
+
+  formattedMaintainers = lib.generators.toPretty {
+    multiline = true;
+    indent = "  ";
```

Comment:
Now you are formatting the whole attrset, I believe we want the default indentation `""`.


Currently it is generating with 4 spaces, then 2:

```nix
{
    "347Online" = {
      email = "katiejanzen@347online.me";
      github = "347Online";
      githubId = 38228372;
      name = "Katie Janzen";
      source = "nixpkgs";
    };
```

```suggestion
```